telegram_bot.py

```diff
@@ -1,44 +1,77 @@
 import requests  # type: ignore
-from typing import Optional
+from typing import Optional, Union
+
+from requests import Response
 from telegram import Update
+from telegram.constants import ParseMode
 from telegram.ext import (
-    filters,
     ApplicationBuilder,
     CallbackContext,
-    CommandHandler,
-    MessageHandler,
 )
+import prettytable as pt
 
 
 class Bot:
-    def __init__(self, api_key: Optional[str], cookies: str, url: str):
+    def __init__(self, api_key: Optional[str]):
         try:
-            self.cookies = cookies
-            self.url = url
             self.application = ApplicationBuilder().token(api_key).build()
-            start_handler = CommandHandler("start", self.start)
-            echo_handler = MessageHandler(filters.TEXT & (~filters.COMMAND), self.echo)
-            self.application.add_handler(CommandHandler("standings", self.standings))
-            self.application.add_handler(start_handler)
-            self.application.add_handler(echo_handler)
+        except Exception as e:
+            print(e)
+
+    def start_bot(self):
+        try:
             self.application.run_polling()
         except Exception as e:
             print(e)
```

Comment:
Do you think that without the bot will be necessary to put `sys.exit(1)` ?

Reply:
The `run_polling` method can raise a `RuntimeError`, so I think we should handle that